### PR TITLE
Chapter 8: Log in

### PR DESCRIPTION
**Pull request by datbk206**
image
Kiến thức cần nắm chương 8:
https://docs.google.com/document/d/1PCx58WedxwzfiQB2sSqu57z8b8CV0vHfIlbdt3Y_-bc/edit?usp=sharing

### CODE REVIEW

**Comment by datbk206** [comment]
ready

File: app/views/sessions/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<% provide :title, t("session.login.title") %>
+<h1><%= t "session.login.title" %></h1>
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for :session, url: login_path do |f| %>
```
Comment (anhnt-1044) [inline app/views/sessions/new.html.erb:5]: a dùng form khác được kh e

Comment (datbk206) [reply app/views/sessions/new.html.erb:5]: có thể dùng form_with ạ

**Comment by datbk206** [comment]
ready

File: app/views/sessions/new.html.erb (hunk 1 above)
Comment (datbk206) [reply app/views/sessions/new.html.erb:5]: <%= form_with scope: :session, url: login_path do |form| %>

**Comment by ngocvt-0484** [comment]
merged

**Pull request merged**